src/components/MenuItem.js

```diff
@@ -1,23 +1,30 @@
 import "./MenuItem.css";
 import useCount from "../hooks/useCount";
+import { BiMinus, BiPlus } from "react-icons/bi";
 
 export default function MenuItem({ item }) {
   const [count, increment, decrement] = useCount();
+
   return (
-    <div className="menu__item">
-      <p>{item.type}</p>
-      <p>{item.name}</p>
-      <p>{item.description}</p>
-      <p>{item.price / 100}</p>
+    <section className="menu__item">
+      <div className="menu__item--content">
+        <img
+          src={`/images/menu/${item.id}.jpg`}
+          alt={item.name}
+          className="menu__item--image"
+        />
+        <p>
+          {item.type}
+          {item.name}
+          {item.description}
+        </p>
+        <p className="menu__item--price">{item.price / 100}</p>
+      </div>
       <div className="menu__counter">
-        <button className="menu__counter--buttons" onClick={increment}>
-          +
```

Comment:
It would be nice for the user experience to have some feedback that the item was indeed added to or reduced from the shopping cart.

Reply:
thanks for the feedback! Hopefully that is coming with the next PR :D